test cases/common/51 pkgconfig-gen/meson.build

```diff
@@ -19,7 +19,7 @@ pkgg.generate(
 )
 
 pkgconfig = find_program('pkg-config', required: false)
-if pkgconfig.found()
+if pkgconfig.found() and build_machine.system() != 'windows'
```

Comment:
This should use the new skipping functionality. Let's merge that first and then this.